numalogic/registry/mlflow_registry.py

```diff
@@ -154,8 +161,11 @@ def load(
                     "Mlflow error when loading a model with key: %s: %r", model_key, mlflow_err
                 )
             return None
+        except ModelVersionError as model_missing_err:
+            _LOGGER.exception("No Model found in %s %r", self.model_stage, model_missing_err)
```

Comment:
```suggestion
            _LOGGER.error("No Model found in %s, error: %r", self.model_stage, model_missing_err)
```
We don't need exception since we are generating the error